docs/telemetry.md

```diff
@@ -54,20 +54,16 @@ The event ping contains a list of events ([see event format on readthedocs.io](h
 | Turbo mode switch clicked              | action   | change                | turbo_mode | on/off (the new value) | |
 | Pin site switch clicked                | action   | change                | pin_page   | on/off (the new value) | desktop_mode* |
 | Desktop mode switch clicked | action | change | desktop_mode | on/off (the new value) | |
-| Tile clicked                           | action   | click                 | home_tile  | bundled/custom | |
+| Tile clicked                           | action   | click                 | home_tile  | bundled/custom/youtube** | |
 | Tile removed                           | action   | remove                | home_tile  | bundled/custom | |
-| Unique tiles clicked per session**     | aggregate| click                 | home_tile  | `<int>` | |
+| Unique tiles clicked per session     | aggregate| click                 | home_tile  | `<int>` | |
 | Menu shown by user              | action   | user_show       | menu         | ||
 | Menu hidden by user              | action   | user_hide       | menu         | ||
 | No menu action taken              | aggregate   | no_action_taken       | menu         | ||
 
 (*)When the pin site switch is clicked, the state (on/off) of the desktop mode switch is also sent.
 
-(**)This is broken: see [#879](https://github.com/mozilla-mobile/firefox-tv/issues/879).** For overlay hidden, we only capture cases where the user closes the overlay themselves:
-- Controller menu button is pressed
-- Controller back button is pressed
-
-e.g. we don't log an event when the overlay is closed automatically after "Home" is selected.
+(**)YouTube tile clicks are a sub-category of bundled tile clicks- each click will count towards both values.
```

Comment:
One more nit: (**) doesn't show up properly in the docs in markup (see the docs/Telemetry.md) - even though it shows up fine in the comments. I think the fix is to wrap it in single quotes so: (`**`) (you can look at the raw text of this by editing it). I tested this by editing the docs/Telemetry.md and looking at preview (even though it's not possible to commit).